databricks/koalas/frame.py

```diff
@@ -3222,6 +3222,113 @@ def merge(left, right: 'DataFrame', how: str = 'inner',
         else:
             return DataFrame(selected_columns)
 
+    def join(left, right: 'DataFrame', on: Optional[Union[str, List[str]]] = None,
```

Comment:
Seems like a mistake. left -> self. I think it's a Python convention.

Reply:
oh, I used `left` and `right` when I updated `merge` function before. I'll submit the PR to fix `merge` function soon.

Reply:
FYI: the PR #479 was merged.